src/LazyMessageReader.ts

```diff
@@ -16,7 +16,7 @@ if (!isLittleEndian) {
   throw new Error("Only Little Endian architectures are supported");
 }
 
-type LazyMessage<T> = T & { toJSON: () => T };
+type LazyMessage<T> = T & { toJSON: () => T; toObject: () => T };
```

Comment:
It occurred to me that we could avoid conflicts with legitimate fields named toJSON by using symbol, i.e. something like `static toObject: unique symbol = new Symbol("toObject")` `[LazyMessage.toObject]: () => T`

Reply:
I wonder if that will work with `JSON.stringify()` - or are you saying we should only do this for the `toObject` field?

Reply:
Sigh, I forgot that the toJSON name is actually used by JSON.stringify 🤦